README.md

```diff
@@ -180,6 +184,9 @@ if (result == MessageDialogResult.Accept)
 }
 ```
 
+It will look like this:
+![](doc/images/destructive.png)
```

Comment:
Have the button say "Delete" instead of "Confirm" :)

Reply:
(The best practice is to avoid buttons that display things like Yes and No. Confirm is pretty close to a Yes.)

Reply:
Done